billing_cost_management/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "is_production" {
+  type        = bool
+  default     = true
```

Comment:
Maybe better to remove the default value, so it fails directly if the variable is not set up